engine/src/main/java/org/terasology/rendering/dag/RenderTaskListGenerator.java

```diff
@@ -177,6 +185,7 @@ private void logIntermediateRendererListForDebugging(List<Node> orderedNodes) {
         return taskList;
     }
 
+    // TODO: Rename method and remove comments
```

Comment:
That seems to have not been removed from back when I thought this method was causing the problems 😄. Removed.